### PR TITLE
Redraw in numberSelectionWidget

### LINKED ISSUE #460: Некорректное выделение элементов в меню "Взаимодействие"
По нажатию на "Enter" не меняется цвет ячейки -- невозможно понять, редактируема ли она сейчас или нет.

### PR DESCRIPTION
**Pull request by khodand**
Fix #460

### CODE REVIEW

**Comment by anastasiia-kornilova** [comment]
Sorry, everything is ok, I have missed something :)

File: trikGui/numberSelectionWidget.cpp [hunk 1]
```diff
@@ -110,7 +110,7 @@ void NumberSelectionWidget::keyPressEvent(QKeyEvent *event)
 	}
 	case Qt::Key_Return: {
 		mEditingMode = !mEditingMode;
-		mWidget->update();
+		this->update();
```
Comment (anastasiia-kornilova) [inline trikGui/numberSelectionWidget.cpp]: `this` is not necessary :)

Comment (khodand) [inline trikGui/numberSelectionWidget.cpp]:
```suggestion
		update();
```

Comment (khodand) [reply trikGui/numberSelectionWidget.cpp]:
```suggestion
		update();
```

**Pull request merged**